cmd/juju/model/defaultscommand_test.go

```diff
@@ -95,16 +230,16 @@ func (s *DefaultsCommandSuite) TestDefaultsInit(c *gc.C) {
 }
 
```

Comment:
please also add tests that verify that the correct cloud/region are passed through to the API calls